examples/cord_ep/README.md

```diff
@@ -0,0 +1,26 @@
+CoRE Resource Directory: Endpoint Example
+=========================================
+
+This example application demonstrates the usage of RIOT's Resource Directory
+(RD) endpoint module, called `cord_ep`. This module supports the registration,
+update, and removal procedures as defined in the
```

Comment:
Wording: "in the draft-ietf-core-…" sound's clunky (might be an artifact from the previous version where it was "in the […] draft".